reflex/reflex.py

```diff
@@ -480,6 +484,11 @@ def deploy(
         "--project",
         help="project id to deploy to",
     ),
+    project_name: Optional[str] = typer.Option(
+        None,
+        "--project-name",
+        help="The name of the project to deploy under.",
```

Comment:
```suggestion
        help="The name of the project to deploy to.",
```

match the sentence structure of the `--project` flag